docs/specs/online_store_format.md

```diff
@@ -92,6 +92,29 @@ Other types of entity keys are not supported in this version of the specificatio
 
 ![Datastore Online Example](datastore_online_example.png)
 
+## Google Bigtable Online Store Format
+
+[Bigtable storage model](https://cloud.google.com/bigtable/docs/overview#storage-model)
+consists of massively scalable tables, with each row keyed by a "row key". The rows in a
+table are stored lexicographically sorted by this row key.
+
+We use the following structure to store feature data in Bigtable:
+
+* All feature data for an entity or a specific group of entities is stored in the same
+  table. The table name is derived by concatenating the lexicographically sorted names
+  of entities.
+* This implementation only uses one column family per table, named `features`.
+* Each row key is created by concatenating a hash derived from the specific entity keys
+  and the name of the feature view. Each row only stores feature values for a specific
+  feature view. This arrangement also means that feature values for a given group of
+  entities are colocated.
+* The columns used in each row are named after the features in the feature view.
+  Bigtable is perfectly content being sparsely populated.
+* By default, we store 1 historical value of each feature value. This can be configured
```

Comment:
seems like the default below is actually 2?

Reply:
The `max_versions` is set to `2`, which I think means one current version and 1 historical version. Or that's what it seems based on [this document](https://cloud.google.com/bigtable/docs/samples/bigtable-create-family-gc-max-versions). Quoted from the document linked above:

```python
    print("Creating column family cf2 with max versions GC rule...")
    # Create a column family with GC policy : most recent N versions
    # where 1 = most recent version

    # Define the GC policy to retain only the most recent 2 versions
    max_versions_rule = column_family.MaxVersionsGCRule(2)

    column_family2 = table.column_family("cf2", max_versions_rule)
    column_family2.create()
    print("Created column family cf2 with Max Versions GC Rule.")
```